api/src/main/java/com/capitalone/dashboard/service/DashboardRemoteServiceImpl.java

```diff
@@ -189,6 +189,11 @@ private Dashboard requestToDashboard(DashboardRemoteRequest request) throws Hygi
             serviceName = service.getConfigurationItem();
         }
         List<String> activeWidgets = new ArrayList<>();
-        return new Dashboard(true, metaData.getTemplate(), metaData.getTitle(), application, metaData.getOwner(), DashboardType.fromString(metaData.getType()), serviceName, appName,activeWidgets, false, ScoreDisplayType.HEADER);
+
+        Dashboard dashboard = new Dashboard(true, metaData.getTemplate(), metaData.getTitle(), application, metaData.getOwner(), DashboardType.fromString(metaData.getType()), serviceName, appName,activeWidgets, false, ScoreDisplayType.HEADER);
+
+        //remoteCreated is actually ignored in Dashboard construction method, so set it again.
+        dashboard.setRemoteCreated(true);
```

Comment:
This is the incorrect place to make this change. It should probably be fixed in the Dashboard.java class as your comment suggest.

Reply:
@liutsi Can you look into the comments above and update this PR?

Reply:
@liutsi can you look into the above PR comments?

Reply:
@liutsi we will end up closing this PR due to inactivity, If you plan to contribute please create a new PR